doc/content/playlist_parsers.md

```diff
@@ -33,6 +35,30 @@ Additionally, you can also manually parse and process a playlist using `request.
 and some programming magic. You can check the code source for `playlist.reloadable` in our standard library
 for a detailed example.
 
+### Remote M3U playlist example
+Here is an example of a m3u playlist being read from nodejs/express .
+
+liquidsoap script:
+```liquidsoap
+#!/usr/local/bin/liquidsoap
+
+settings.http.mime.extnames := [...settings.http.mime.extnames(), ("audio/x-mpegurl", ".m3u")]
+p = playlist(reload=10, "http://localhost:8080/radio/playlists/0/playlist.m3u")
+```
+
+nodejs/express app:
+```js
+import express from "express";
+const app = express();
+app.get('/radio/playlists/:id/playlist.m3u', async (req, res) => {
+  const playlist = ["/media/foo.mp3", "/media/bar.mp3"]
+  res.set('Content-Type', 'audio/x-mpegurl')
+  res.set(`Content-Disposition: attachment; filename="playlist-${req.params.id}.m3u`);
```

Comment:
```suggestion
  // Liquidsoap will use the file extension from the `Content-Disposition` to guess the
  // playlist format
  res.set(`Content-Disposition: attachment; filename="playlist-${req.params.id}.m3u`);
  
  // Otherwise, it will try to guess the file extension from the playlist mime-type.
  res.set('Content-Type', 'audio/x-mpegurl')
```